var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -25,7 +25,9 @@ class Pango(AutotoolsPackage):
 
     depends_on("pkgconfig", type="build")
     depends_on("harfbuzz")
-    depends_on("cairo")
+    depends_on("freetype")
+    depends_on("fontconfig")
```

Comment:
Are freetype and fontconfig _direct_ dependencies of pango, or is `cairo+ft+fc` the only thing that is needed here?

Reply:
@adamjstewart, you are correct, cairo+ft+fc will suffice since freetype and fontconfig will be picked up from the cairo dependence. I removed their dependences from the latest commit.